docs/integrations/plugins-101-building-your-first-plugin/why-build-plugins.md

```diff
@@ -0,0 +1,29 @@
+# Why Build Plugins? Introduction to the Value and Impact of Plugins within Backstage
+
+Backstage plugins are essential components that enable the integration of various tools and services into a unified developer portal. Here’s a detailed look at why building plugins for Backstage can be highly beneficial:
```

Comment:
This kind of reads like its about to go into bullet points like beneficial because:

- one
- two
- three

Maybe we can reword it to something like, below we list out the reasons of why building backstage plugins is beneficial

WDYT?

Reply:
I updated the headers below with numbers to make the list structure a bit clearer - I'd like to take a second pass on this and figure out the narrative we're looking for for the guides as I think it determines what things we put here, but I think this is a good start.

Reply:
This format works for me :+1: